phpdotnet/phd/Package/PHP/Web.php

```diff
@@ -254,10 +254,21 @@ protected function writeJsonIndex() {
      * used to generate the search index and the descriptions JSON files.
      */
     private function processJsonIndex(): array {
+        $alwaysIncludeElements = [
+            'refentry',
+            'stream_wrapper',
+            'phpdoc:classref',
+            'phpdoc:exceptionref',
+            'phpdoc:varentry',
+        ];
+
         $entries = [];
         $descriptions = [];
         foreach($this->indexes as $id => $index) {
-            if (!$index["chunk"]) {
+            if (
+                (! in_array($index['element'], $alwaysIncludeElements, true))
+                && (! $index['chunk'])
+            ) {
```

Comment:
I think having the `!index['chunk']` check first makes more sense as it should be quicker and also the most common reason for performing the second check?